cmd/cel-shed/datastore.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ipfs/boxo/blockstore"
+	ds "github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+	dsq "github.com/ipfs/go-datastore/query"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+)
+
+func init() {
+	datastoreCmd.AddCommand(eraseCmd, eraseSamplesCmd)
+}
+
+var datastoreCmd = &cobra.Command{
+	Use:   "datastore [subcommand]",
+	Short: "Collection of datastore related utilities",
+}
+
+var eraseCmd = &cobra.Command{
+	Use:   "erase <ds_key>",
```

Comment:
Why need the discover stopped? Why can't they pass either `--node.store` or `--p2p.network` to make it more explicit and versatile cmd?

Reply:
It should have both, so you can be lazy without specifying anything, but also can override of non default path

Reply:
I personally have multiple types of nodes for multiple networks. Would appreciate an option to specify network name + type of the node in this command. And only network for command of erasing samples

Reply:
Yes, that's TBD